WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -175,17 +174,22 @@ public void onClick(View v) {
                 }
             });
         } else {
-            // Remove the selectableItemBackground if the user can't be edited
-            if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.JELLY_BEAN) {
-                mRoleContainer.setBackgroundDrawable(null);
-            } else {
-                mRoleContainer.setBackground(null);
-            }
+            clearRoleContainerBackground();
             // Change transparency to give a visual cue to the user that it's disabled
             mRoleContainer.setAlpha(0.5f);
         }
     }
 
+    @SuppressWarnings("deprecation")
+    // Remove the selectableItemBackground if the user can't be edited
```

Comment:
May I suggest reinstating this comment back to the line the function is called? The function name is pretty clear and perfectly corresponding to its body, whereas the "if the user can't be edited" part is not implemented here but rather, at the caller's position. Sorry for the mind-blowing nitpicking! 😃 ❤️